sql/CtsSchema.dbschema

```diff
@@ -41922,7 +41922,6 @@
                       <element>
                         <OBJECT REFERENCE="com.sun.forte4j.modules.dbmodel.TableElement-1952968163"/>
                       </element>
-                      <table>INTEROP_CSIV2_TABLE1</table>
                       <keys>
```

Comment:
@gurunrao @RohitKumarJain @alwin-joseph @bhatpmk 

Since we can still run the Platform TCK with the INTEROP entries removed, this pull request is probably okay.  I'm going to give my approval but for undestanding more, if anyone has answers to the below, please do respond.

I see that install/websocket/bin/xml/impl/glassfish/deploy.xml + bin/xml/impl/glassfish/deploy.xml use it only if sun-cmp-mappings.xml file is found, I think like ./src/com/sun/ts/tests/ejb/ee/pm/ejbql/schema/ejb_ECmpQL_schema_ejb.jar.sun-cmp-mappings.xml

Was sql/CtsSchema.dbschema generated by a tool and checked in?